Geofencer/Geofencer/PlacesDecoder.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PlacesDecoder.swift
+//  Geofencer
+//
+//  Created by Brett Parsekian on 8/23/19.
+//  Copyright © 2019 bparsekian. All rights reserved.
+//
+
+import Foundation
+
+struct PlacesDecoder {
+    static let filename = "Saved Places"
+    static let fileExt = "json"
+    
+    static func fetchPlaces() -> [Place] {
```

Comment:
Always tons of debate and hard opinions around this, but I'd personally prefer not to implement this functionality with static variables/methods.  I'd most likely implement ```PlacesDecoder``` as a class instead of a struct, and inject an instance of the class wherever it's needed (probably in a constructor for ```PlacesTableViewController``` in this case).  We can talk more about dependency injection and how it relates to this scenario, including testing considerations, next week